Makefile

```diff
@@ -4,13 +4,13 @@ all: format check requirements.txt
 
 format: prettier isort black
 
-check: mypy checkprettier checkisort checkblack
+check: mypy pyright checkprettier checkisort checkblack pylint flake8
```

Comment:
- maybe all the `python` checks should be first and only then the `JS` check (`checkprettier`)

Reply:
8bf41035